near-sdk/src/utils/stable_map.rs

```diff
@@ -40,4 +41,12 @@ impl<K, V> StableMap<K, V> {
     pub(crate) fn inner(&mut self) -> &mut BTreeMap<K, Box<V>> {
         self.map.get_mut()
     }
+    pub(crate) fn with_value<Q: ?Sized, F, T>(&self, k: &Q, f: F) -> Option<T>
```

Comment:
My thinking was that this is not actually mapping the value since it's just calling a function on the reference to the value, not moving the owned value to be mapped. This might just be semantics, but map would be `FnOnce(V) -> T` when this is actually `FnOnce(&V) -> T`

If you feel strongly about this I'm happy to change since I don't really have an opinion here and it's a private API

Reply:
ahh I see. I don't have too strong of an opinion what this should be either. And huh, [VaListImpl::with_copy](https://doc.rust-lang.org/std/ffi/struct.VaListImpl.html#method.with_copy) has a similar signature, so it's not too bad for it to have a prefix of `with_` but certainly very uncommon

Reply:
renamed to `map_value_ref`. I don't really think there is a great name, but it's internal and this might be the clearest. Sound good to you?

Reply:
Yup that sounds good to me. Thanks